openmdao/approximation_schemes/complex_step.py

```diff
@@ -214,26 +221,21 @@ def _run_point_complex(self, system, input_deltas, out_tmp, result_clone, deriv_
         elif deriv_type == 'partial':
             run_model = system.run_apply_nonlinear
             results_vec = system._residuals
-        else:
-            raise ValueError('deriv_type must be one of "total" or "partial"')
```

Comment:
It seems like it would be more robust and wouldn't hurt to leave the ValueError in there... I know this is not normally called by the user, but it's looks like incomplete logic to not handle the case where neither the `if` nor the `elif` is true and just fall through (leaving an opening for future bugs to creep in, e.g. if someone typos the arg)...

Reply:
see above